src/TogglePassword/assets/src/style.css

```diff
@@ -0,0 +1,28 @@
+.toggle-password-container {
+    position: relative;
+}
+.toggle-password-icon {
+    height: 1rem;
+    width: 1rem;
+}
+.toggle-password-button {
+    align-items: center;
+    background-color: transparent;
+    border: none;
+    color: #3b82f6;
+    column-gap: 0.25rem;
+    cursor: pointer;
+    display: flex;
+    flex-direction: row;
+    font-size: 0.875rem;
+    justify-items: center;
+    height: 1rem;
+    line-height: 1.25rem;
+    position: absolute;
+    right: 0.5rem;
+    top: -1.25rem;
```

Comment:
This is going to be the trickiest part of this PR. Can we make some CSS that works in "most" cases. I was testing locally on https://ux.symfony.com/live-component/demos/auto-validating-form

I see the form theme template, which contains the `toggle-password-container` class, which is the key to making this all work "out of the box".  I also see that you prepended the config to add this form theme automatically - smart! Unfortunately... because ux.symfony.com already uses a form theme, by prepending the config, the custom `bootstrap_5_layout.html.twig` from that project shows up last in the list, and so its form theme wins (and the one from this bundle isn't used). I also think that there should probably be a way to "opt out" of the form theme - you may want to use your normal `password_widget` rendering (without the new div) and then use your own CSS to place things correctly.

I'm not convinced I have the correct solution for this. But my idea is:

A) Remove the prepend and instead document the user adding the form theme manually. It kinda sucks to do this... but then the user can opt out of it.

B) Document that, instead of the form theme, you can also style manually.

Another idea, might be to:

1) In `TogglePasswordTypeExtension`, add a new `toggle_form_theme` option, which defaults to `true`.
2) In `TogglePasswordTypeExtension`, if that's true, add a new item to the `block_prefixes` variable - like this https://github.com/symfony/ux/blob/24112fa46bb7e192d69de1b1e5f4365186bb296b/src/Autocomplete/src/Form/ParentEntityAutocompleteType.php#L56 - something like `ux_password_toggle`.
3) Change the form theme template to override *just* that block - e.g. `ux_password_toggle_widget`.

The idea would be that, if `toggle_form_theme` is true, then it will use your form theme. If false, it will skip and use the normal. Additionally, in your form theme (`form_theme.html.twig`), I believe you could then:

```diff
-{ block('form_widget') }}
+{ block('password_widget') }}
```

So in the event the user is overriding the `password_widget` in a custom form theme, we will still use that.

Also, we should document how this all works a bit - basically communicating:

A) There is a custom form theme that's activated, which wraps your widget in a `<div class="toggle-password-container">`.
B) You can turn this off with the `toggle_form_theme` option, but then you're on your own to style.

Reply:
Nice catch ! Indeed, that's tricky but absolutely relevant !!
I'd go for your second suggestion, adding a toggle_form_theme boolean option, which would define a block prefix 👍🏼 
Let's go for that 😄

Reply:
Ok, done ! 
I went for `use_toggle_form_theme` option, with `true` as default value so it can add a custom block prefix : `ux_toggle_password` (to keep consistency with the names used everywhere else in the package) and i add documentation about the behaviour to expect.
I tried to be as clear as possible on this one while while remaining as concise as possible. I hope it's clear enough, otherwise i'll try another way 😄